### ISSUE TITLE
Fog nfs local blobstore not compatible with tls

### ISSUE THREAD

**Issue by youngm** [cloudfoundry/cloud_controller_ng#893]
## Issue

Enabling `temporary_droplet_download_mtls` causes droplet downloads to fail for my local nfs blobstore.

## Context

I enabled `temporary_droplet_download_mtls` and noticed that droplet downloads were failing the the rep and CC with a 404.  Error messages looked like:

```
[error] 22895#0: *691 open() "/var/vcap/packages/nginx/html/cc-droplets/d3/f3/d3f3aea7-1cab-4296-ba36-26dc60027006/86495f15232be36d99b8741957373ebb327b848c" failed (2: No such file or directory), client: 10.118.216.131, server: _, request: "GET /internal/v4/droplets/a679af1a-9d67-4bd0-b61f-762174ea9fff/bfb389648873b1d2571c3ab38d7edae318efed44fe0a08fbfc8f9cc907408943/download HTTP/1.1", upstream: "http://unix:/var/vcap/sys/run/cloud_controller_ng/cloud_controller.sock/internal/v4/droplets/a679af1a-9d67-4bd0-b61f-762174ea9fff/bfb389648873b1d2571c3ab38d7edae318efed44fe0a08fbfc8f9cc907408943/download", host: "cloud-controller-ng.service.cf.internal:9023", referrer: "http://cloud-controller-ng.service.cf.internal:9022/internal/v2/droplets/a679af1a-9d67-4bd0-b61f-762174ea9fff/bfb389648873b1d2571c3ab38d7edae318efed44fe0a08fbfc8f9cc907408943/download"
```

## Steps to Reproduce

* Enable local nfs blobstore in cc
* Enable mtls `temporary_droplet_download_mtls`
* Note that you cannot download droplets using the tls port.

## Expected result

Be able to download droplets on the tls port.

## Current result

404 on download droplet requests.

## Possible Fix

Add location config into nginx tls server.

**Comment by tcdowney** [comment]
Thanks for reporting this @youngm, we were able to reproduce it in one of our environments and we believe the code change in 42422bc9386923139ab0754c0b52cbea96ba3acd should fix it.

**Comment by youngm** [comment]
Great!  Thanks @tcdowney

**Comment by zrob** [comment]
fix should be available in capi release 1.42.0
